src/hooks/usePreventBodyScroll.ts

```diff
@@ -0,0 +1,33 @@
+import { useIsomorphicEffect } from './useIsomorphicEffect';
+
+/**
```

Comment:
This JSDoc documentation should be present right above the function `usePreventBodyScroll`. So at line number 18.